### DISCUSSION TITLE
Thank you. Please help me resolve the error message

### DISCUSSION THREAD

**Discussion by PeterHYJang** [scholarly-python-package/scholarly#530] [Q&A]
I am a beginner using Python in Visual Studio Code.

I installed scholarly using the following command:
`**`pip3 install scholarly`**`

When I run the Example Code, I encounter the following error message. Could you please advise on how to resolve this issue?"

**Exception has occurred: ImportError
cannot import name 'formatargspec' from 'inspect' (C:\Python311\Lib\inspect.py)
  File "C:\Users\Joonha\Desktop\Python_WS\test_file.py", line 1, in <module>
    from scholarly import scholarly
ImportError: cannot import name 'formatargspec' from 'inspect' (C:\Python311\Lib\inspect.py)**

**Comment by arunkannawadi** [comment]
What version of python and `inspect` do you have? And is this the full traceback?
